docs/release_notes.rst

```diff
@@ -193,6 +226,14 @@ Analysis
   the :ref:`generators_eia860` table so associating those gf and bf records are more
   cleanly associated with generators. Thanks to :user:`grgmiller` for his
   contribution, which was integrated by :user:`cmgosnell`! See PRs :pr:`2235,2446`.
+* The :mod:`pudl.analysis.mcoe` table now uses the allocated estimates for per-generator
```

Comment:
Do these bullets about the new MCOE calculations and tables belong under this analysis section? 

What else am I missing to add to the release notes?

Reply:
IIRC when I put down the MCOE PR I had just started playing with using the new allocated generation + fuel outputs to calculate heat rates, and **had not done any validation of the outputs** so this might be dangerous.  What kind of sanity checking have you done?  Does it still pass the data validation tests?  How do the old vs. new distributions compare when broken down by the type of unit (combustion turbine, combined cycle, and steam turbine)?

We should compare the outputs from the old and new methods, including the overall coverage, the restricted outputs (where we only include reasonable values) and the unrestricted outputs (where crazy outputs are still present) to see what the effects of this change are.